src/static/js/pad_impexp.js

```diff
@@ -243,8 +243,19 @@ var padimpexp = (function()
       if (status !== "ok")
       {
         importFailed(status);
+      }else{
+        $('#import_export').removeClass('popup-show');
+      }
+
+      // directDatabaseAccess is a string not a boolean
+      if(directDatabaseAccess !== 'undefined'){
+        // commented out for now.  This needs a proper fix after 1.8.3
+        // TODO: pad.switchToPad() should not require a reload of the page.
+        // pad.switchToPad(clientVars.padId);
+        location.reload();
+        // Switches to the pad without redrawing the page
+        $('#import_export').removeClass('popup-show');
       }
-      if(directDatabaseAccess) pad.switchToPad(clientVars.padId);
```

Comment:
If `directDatabaseAccess` was a literal `"undefined"` string, this means that this instruction should had never fired.

Is this the case?

Reply:
It worked at one point, something must have changed it to break it.  I am kinda surprised by it too..